workers/unity/Assets/Playground/Config/PlayerTemplate.cs

```diff
@@ -32,5 +36,57 @@ public static EntityTemplate CreatePlayerEntityTemplate(string workerId, Improba
 
             return entityBuilder.Build();
         }
+
+        private static void MinimapExample()
+        {
+            var playerConstraint = Interest
+                .Query(Constraint.All(
+                    Constraint.RelativeSphere(20), 
+                    Constraint.Component<PlayerInfo.Component>()))
+                .Result<Position.Component, PlayerInfo.Component>();
+
+            var miniMapConstraint = Interest
+                .Query(Constraint.All(
+                    Constraint.RelativeBox(50, double.PositiveInfinity, 50),
+                    Constraint.Component<MinimapRepresentation.Component>()))
+                .Result<Position.Component, MinimapRepresentation.Component>();
+
+            var interest = new InterestComponent()
+                .AddQuery<PlayerControls.Component>(playerConstraint)
+                .AddQuery<PlayerControls.Component>(miniMapConstraint);
+        }
+        
+        private static void TeamsExample()
+        {
+            var isblue = true;
+            //some logic to determine which team
+
+            var teamQuery = Interest
+                .Query(Constraint.Component(isblue ? BlueTeam.ComponentId : RedTeam.ComponentId));
+            
+            var interest = new InterestComponent()
+                .AddQuery<PlayerControls.Component>(teamQuery);
+        }
+
+        private static void FrequencyExample()
+        {
+            var playerConstraint = Interest
+                .Query(Constraint.All(
+                    Constraint.RelativeSphere(20), 
+                    Constraint.Component<PlayerInfo.Component>()))
+                .Frequency(20)
```

Comment:
Is this in Hz?

Reply:
yes it's Hz - could probably rename it something like `FrequencyHz` to make that clearer

also plan on adding docs/code comments to explain a lot of this further

Reply:
Always put the units in the name if applicable. MaxFrequencyHz would be my preference (although Hz is the only bit I feel strongly about).